Projects/Luym_Building/style.css

```diff
@@ -31,6 +31,10 @@
 body {
   font-family: "MinecraftiaRegular";
   color: black;
+
+  /* main background */
+  background: url(/Assets/Images/background/bg_whitetown.jpg) no-repeat center center fixed;
+  background-size: cover;
```

Comment:
someting to reminds you here
I realize JQuery animation easing wont works with fixed background since it will fix page scrolling
gotta find another way to put background if you want more cool easing in the future

> https://user-images.githubusercontent.com/77855014/139573826-24d11e44-a0d4-45f1-912c-2cb22f43da7b.mp4

Reply:
Hi @tintinkung, I'm having difficulty understanding what you're trying to show me in the video sorry! 

Did you remove the background for the Mandalay project because it was causing a fixed scrolling issue? Or are did the background disappear in the second browser?

Reply:
i edited nothing, that was just the old page when easing works

Reply:
when theres fixed background easing become linear constantly

Reply:
Looks like it can be cache issue perhaps, just tested it on Chrome, Firefox and Safari on my computer and it's working as intended. Can you try clearing your browser cache and retry? The way the easing works on the second browser in the video works the same for me with fixed background on all my current browsers.

Reply:
_Sorry for the quality, my Macbook renders videos too high size so I had to export lower for upload._

My easing is working well with fixed background.

https://user-images.githubusercontent.com/8276671/139575220-61f15cf9-67c2-4620-ba93-d3715e7408c0.mov

Reply:
hm looks like i cant get it to works for me it always turn out to be very linear ease. 
maybe i'll blame my pc sorry

Reply:
No worries,  I'm sure we can find a fix. Can others test this on their pc to see if it's a group issue or individual?

If it's a group issue I'll find a fix for it.

Reply:
I tried debugging I think the actual easing does it work, the only pre-ease action seems to happen randomly for me

https://user-images.githubusercontent.com/77855014/139576178-0fd0316b-e1ff-4d0f-ad06-ec5e6eb3be9a.mp4

Reply:
That's strange, I'll definitely look into it. If others come across the same issue that wouldn't be good, maybe I'll have to find a different way to add the easing. I'll also do some debugging on my end to see if I can get the issue you're getting.

If you can get your team to try and see if they get the same issue that would be great, we can find out if it's a group issue or not.